package.json

```diff
@@ -42,7 +42,7 @@
             "node_modules/",
             "!node_modules/pc-nrfjprog-js/nrfjprog",
             "resources/*.html",
-            "resources/*.png",
+            "resources/icon.*",
```

Comment:
I believe the `resources/*.png` rule still needs to be there. If not, the splashscreen.png will not be included in the artifact.

Reply:
Oh, yes, you are right. And I see the resolution of `splashscreen.png` is ok but can be better. This is another issue though.